runtime/rococo/src/lib.rs

```diff
@@ -1645,7 +1645,7 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
-	#[api_version(3)]
+	#[api_version(4)]
```

Comment:
I don't understand this change. This changes `disputes` to version 4, while elsewhere it's still marked as version 3, e.g. in `primitives/src/runtime_api.rs`. But I feel like I'm missing context.

Reply:
No, this line actually says 'this runtime implements version 4 of the `ParachainHost` api'. It's not related just to the dispute. The result in the context of this PR is that rococo runtime will include the new methods introduced here.

It's not related to a single method, but for the whole implementation. I think you are confused because `api_version` is used in two different contexts:
- in `decl_runtime_apis` macro -  It decorates just a single method there and specifies in which runtime version it has been included.
- in `impl_runtime_apis` macro - It decorates an impl block and indicates which version of the api this runtime implements.

Does this make sense?

Reply:
Yes, thank you! 😅 I am used to a bump in the version number meaning "this will be introduced later", whereas here it's the opposite.

Reply:
Do you think we can make it less confusing somehow? E.g. by renaming `api_version` in the impl block to `impl_version` or `api_impl_version`?

Reply:
If possible, that would be great, and would likely avoid confusion in the future. I like `api_impl_version` -- explicit and clear.